bin/next-ver.js

```diff
@@ -39,7 +39,24 @@ function setVersion (newVersion) {
   })
 }
 
+function printVersion (nextVersion) {
+  if (nextVersion && !args.silent) {
+    console.log('next version should be', nextVersion)
+  }
+
+  if (nextVersion && args.silent) {
+    console.log(nextVersion)
+  }
+
+  if (!nextVersion) {
+    console.log('no new version judging by commits')
+  }
+
+  return nextVersion
+}
+
 decideStartVersion()
   .then(computeNextVersion)
+  .then(printVersion)
```

Comment:
you can use `R.tap(printVersion)` here to avoid returning `nextVersion` in that case the `printVersion` could be much simpler
```js
function printVersion(nextVersion) {
  if (!nextVersion) {
    console.log('no new version judging by commits')
    return
  }
  if (args.silent) {
    console.log(nextVersion)
  } else {
    console.log('next version should be', nextVersion)
  }
}
```